recipes/tardis/meta.yaml

```diff
@@ -0,0 +1,107 @@
+{% set name = "tardis-sn" %}
+{% set version = "2.0.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  git_url: https://github.com/tardis-sn/tardis.git
+  git_branch: master
+
+build:
+  number: 0
+  skip: true  # [not win]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - setuptools
```

Comment:
Are all of these dependencies needed when installing, I think you probably only need:
```yaml
    - python >= 3
    - pip
    - astropy =3
    - setuptools
```
If you also need `cython` you should add a compilers to the _**build**_ requirements:
```yaml
  build:
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```

Reply:
This was my first approach, but I don't know why it was failing to compile. So I put all that boilerplate code. I'm pretty sure I need at least one more package in host section, but I don't know which one.

Reply:
For the host section it's generally easier to start with a small list and then build up from there. If you get stuck with an error you don't understand ping me and I'll take a look.